src/main/java/org/mockbukkit/mockbukkit/entity/EntityMock.java

```diff
@@ -1584,12 +1584,10 @@ public void lookAt(double v, double v1, double v2, @NotNull LookAnchor lookAncho
 
 	public void tick()
 	{
-		if (isDead())
+		if (!isDead())
 		{
-			return;
+			++ticksLived;
 		}
-
-		++ticksLived;
 	}
```

Comment:
This is a no-op change. Is this really necessary to add to this pr? Like this is more of a weak preference, doesn't really matter does it.

Reply:
I really think it does not matters, generally the previous behavior is used to un-nest a series of nested if statements, but this does not matter for such small amount of code. I think every programmer will understand this anyhow

Reply:
Shall I revert?

Reply:
Nope, it does not matter, it just does not do anything